SPECS/kernel/config

```diff
@@ -7374,7 +7375,6 @@ CONFIG_IPE_PROP_DM_VERITY=y
 CONFIG_IPE_PROP_DM_VERITY_SIGNATURE=y
 CONFIG_IPE_PROP_FS_VERITY=y
 CONFIG_IPE_PROP_FS_VERITY_BUILTIN_SIG=y
-CONFIG_IPE_PROP_INTENDED_PATHNAME=y
```

Comment:
"IPE_PROP_INTENDED_PATHNAME" is no longer included in the AzL kernel ipe source to better align with upstream offerings. Commit was removed in feature branch: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/f87bb1279e1c0177f804b363c413a281856da12d#diff-1f7bd1d89ce31840a8be21b3700fda99d047643f5a1c614c6dbb00ec86bf9341R12

for reference, commit which contained this feature was [d356a4ff68] ipe: add intended_pathname property